### ISSUE TITLE
File Handles

### ISSUE THREAD

**Issue by edxp** [Exslims/MercuryTrade#216]
Are you absolutely sure you need 825 handles open for `client.txt` log file?

I mean come on https://gist.github.com/anonymous/ccd6c5cc59c8c8cac260680d3d3a5d4b

**Comment by Exslims** [comment]
always 825 handlers?

**Comment by edxp** [comment]
No, right now it is at 1335 https://gist.github.com/anonymous/6773a67be4245f9f607ebd2431ce3d6a
This results in a heavy system load if app crashes or is being closed. Namely the entire system can hang for seconds if it crashes.
